src/help/zaphelp/contents/releases/2_7_0.html

```diff
@@ -98,6 +98,10 @@
 
 Gets the path to ZAP's home directory.
 
+<H3>VIEW localProxies / additionalProxies</H3>
+
```

Comment:
This blank can be removed? (Consistent with other entries)

Reply:
Theres actually a mixture in the rest of the file, so I've removed the other ones with extra lines as I think that makes the source easier to read.

Reply:
Ok